layouts/partials/site-header.html

```diff
@@ -53,9 +53,9 @@ <h1 class="font-bold text-3xl md:text-5xl">{{ . | markdownify }}</h1>
 
         <div class="relative max-w-7xl mx-auto flex items-center justify-between text-white px-4 sm:px-6">
             <div class="lg:w-2/3 text-3x">
-                <div class="flex items-center mt-10 lg:mt-40">
+                <div class="flex items-center mt-10 md:mt-40">
```

Comment:
This change and the one below were made so that users may see the menu on medium-sized screens.